### ISSUE TITLE
Delay in closing the actions popup when selecting "Reply in thread"

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#20987]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open a chat
2. Send a message
3. Right-click on the message
4. Click on "Reply in thread"

## Expected Result: 
The actions popup should close immediately upon selecting "Reply in thread"
## Actual Result: 
There is a noticeable delay in the closure of the actions popup when clicking on "Reply in thread"

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.29-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/a76ccbc5-5074-4054-ada1-230788196ee3

https://github.com/Expensify/App/assets/43996225/530b79e9-e1a5-4388-b41f-c6f749c11bda

**Expensify/Expensify Issue URL:**
**Issue reported by:** @ayazhussain79 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1686348571158339

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by bernhardoj** [comment]
This issue https://github.com/Expensify/App/issues/19776 will handle the delay

**Comment by bfitzexpensify** [comment]
Agreed, the solution in https://github.com/Expensify/App/issues/19776 will address this issue. Think we can go ahead and close this.
